### ISSUE TITLE
Gulpfile: unexpected token

### ISSUE THREAD

**Issue by Toomean** [yeoman/generator-webapp#749]
There is an error inside gulpfile.babel.js
```
gulpfile.babel.js: Unexpected token (56:1)

  54 |   return lintBase('app/scripts/**/*.js')
  55 |     .pipe(dest('app/scripts'));
> 56 | });
     |  ^
  57 | function lintTest() {
  58 |   return lintBase('test/spec/**/*.js')
  59 |     .pipe(dest('test/spec'));
```

**Comment by silvenon** [comment]
Thanks for reporting. There are two extra brackets in lint functions, just remove those and you will be fine. Will publish a fix tomorrow.

**Comment by ghost** [comment]
@silvenon  I was just about to post the solution to this issue. I am relatively new to programming, it took me a few hours to figure this issue out. Once I took the time to actually read the errors it was easy to find the bug.

**Comment by silvenon** [comment]
Should be fixed in [v4.0.0-1](https://github.com/yeoman/generator-webapp/releases/tag/v4.0.0-1). 🚀
